### DISCUSSION TITLE
How can add a new chain to walletConnect supporting chains list

### DISCUSSION THREAD

**Discussion by feiyuanwork** [WalletConnect/walletconnect-monorepo#838] [Developer Support] [answered]
I am a developer of a new chain in Japan. We are planning to use walletConnect on our wallet and marketPlace. Would you tell me how to add a new chain to walletConnect supporting chains list?

**Comment by Seifollahy** [comment]
seifollahy

**Reply by ranjith-edx** [reply]
did you got answer because i'm also searching for same answer

**Comment by bkrem** [comment, marked as answer]
Hi @feiyuanwork,

We now have an issue template that specifies what is needed to request for a chain to be added: https://github.com/WalletConnect/walletconnect-monorepo/issues/new/choose

<img width="800" alt="Screenshot 2023-01-05 at 16 37 50" src="https://user-images.githubusercontent.com/8663099/210820405-db514a0b-5ab7-4ab0-9015-8ec866f5fbdb.png">

cc @finessevanes

**Reply by ranjith-edx** [reply]
hi it says to submit the following details i have the details but exactly where i need to submit it ..?!

**Reply by finessevanes** [reply]
https://github.com/WalletConnect/walletconnect-monorepo/issues/new?assignees=&labels=type%3A+new+chain+request&template=new_chain_to_explorer.md&title=

Is there a section you're unsure about?

**Reply by ranjith-edx** [reply]
oh ok it should be directly submitted as issue...! Got it

**Reply by ranjith-edx** [reply]
or PR

**Reply by finessevanes** [reply]
As an issue please :)
